src/Tribe/Linked_Posts.php

```diff
@@ -457,13 +487,18 @@ public function get_linked_post_types() {
 	/**
 	 * Get Linked Post info
 	 *
-	 * @param string    $linked_post_type Post type of linked post
-	 * @param array     $args             Extra WP Query args.
-	 * @param array|int $linked_post_id   Post ID(s).
+	 * @since TBD Added the $return_all_if_none parameter.
+	 *
+	 * @param string    $linked_post_type   Post type of linked post
+	 * @param array     $args               Extra WP Query args.
+	 * @param array|int $linked_post_id     Post ID(s).
+	 * @param bool      $return_all_if_none True if you want all posts returned if none are found
+	 *                                      (e.g. creating a drop-down). False if you want none
+	 *                                      returned if none are found.
 	 *
 	 * @return array
 	 */
-	public function get_linked_post_info( $linked_post_type, $args = array(), $linked_post_ids = null ) {
+	public function get_linked_post_info( $linked_post_type, $args = array(), $linked_post_ids = null, $return_all_if_none = false ) {
```

Comment:
If we want to avoid using an extra parameter here, you could feasibly just add a filter instead:

```php
/**
 * blah blah
 */
$return_all_if_none = apply_filters( 'tribe_events_return_all_linked_posts_if_none', false, $linked_post_type, $args, $linked_post_ids );
```

Then when you call this method for that one case where you want that to be true, just do:

```php
add_filter( 'tribe_events_return_all_linked_posts_if_none', '__return_true' );
// do the call
remove_filter(  'tribe_events_return_all_linked_posts_if_none', '__return_true' );
```